avocado_vt/plugins/vt_init.py

```diff
@@ -177,8 +234,26 @@ def initialize(self):
                                      help_msg=help_msg, key_type=bool,
                                      default=False)
 
+            # [vt.filter] section
+            help_msg = ("Allows to selectively skip certain default filters. "
+                        "This uses directly 'tests-shared.cfg' and instead of "
+                        "'$provider/tests.cfg' and applies following lists of "
+                        "default filters, unless they are specified as "
+                        "arguments: no_9p_export,no_virtio_rng,"
+                        "no_pci_assignable,smallpages,default_bios,ridge,"
+                        "image_backend,multihost. This can be used to eg. run "
+                        "hugepages tests by filtering 'smallpages' via this "
+                        "option.")
+            settings.register_option('vt.filter', key='default_filters',
+                                     nargs='+', default=None,
+                                     help_msg=help_msg)
+
             # [plugins.vtjoblock] section
             help_msg = 'Directory in which to write the lock file'
             settings.register_option('plugins.vtjoblock', 'dir',
                                      help_msg=help_msg,
                                      default='/tmp')
+
+            virt_loader = getattr(importlib.import_module('avocado_vt.loader'),
```

Comment:
I'm not following the reason to use `importlib` here.  Why not simply:

```python
from ..loader import VirtTestLoader
from avocado.core import loader

loader.register_plugin(VirtTestLoader)
```

Reply:
Unfortunately I can't do global import here. Because the `VirtTestLoader` have to be registered after initialization because of dependencies. And I think that the local import is better via `importlib`.